scripts/us_epa/superfund/sites/tar_creek/process_contaminants.py

```diff
@@ -0,0 +1,240 @@
+import os
+import numpy as np
+import pandas as pd
+from absl import app, flags
+
+_UNIT_MAP = {
+    "Cond.": "dcs:MicroseimensPerCentimeter",
+    "pH": "",
+    "Hardness": "dcs:MilligramsCaCO3PerLitre",
+    "temp": "dcs:Celsius",
+    "D.O.": "dcs:MilligramsPerLitre",
+    "Sulfate": "dcs:MilligramsPerLitre",
+    "Iron": "dcs:MilligramsPerLitre",
+    "Lead": "dcs:MilligramsPerLitre",
+    "Zinc": "dcs:MilligramsPerLitre",
+    "Cadmium": "dcs:MilligramsPerLitre"
+}
+
+_BASE_SV_MAP = {
+    "Cond.": {
+        "populationType": "BodyOfWater",
+        "waterSource": "GroundWater",
+        "measuredProperty": "electricalConductivity",
+    },
+    "pH": {
+        "populationType": "BodyOfWater",
+        "waterSource": "GroundWater",
+        "measuredProperty": "potentialOfHydrogen"
+    },
+    "Hardness": {
+        "populationType": "BodyOfWater",
+        "waterSource": "GroundWater",
+        "measuredProperty": "waterHardness"
+    },
+    "temp": {
+        "populationType": "BodyOfWater",
+        "waterSource": "GroundWater",
+        "measuredProperty": "temperature"
+    },
+    "D.O.": {
+        "populationType": "BodyOfWater",
+        "waterSource": "GroundWater",
+        "solute": "Oxygen",
+        "solvent": "Water",
+        "measuredProperty": "concentration"
+    },
+    "Sulfate": {
+        "populationType": "BodyOfWater",
+        "contaminatedThing": "GroundWater",
+        "contaminant": "Sulfate",
+        "measuredProperty": "concentration"
+    },
+    "Iron": {
+        "populationType": "BodyOfWater",
+        "contaminatedThing": "GroundWater",
+        "contaminant": "Iron",
+        "measuredProperty": "concentration"
+    },
+    "Lead": {
+        "populationType": "BodyOfWater",
+        "contaminatedThing": "GroundWater",
+        "contaminant": "Lead",
+        "measuredProperty": "concentration"
+    },
+    "Zinc": {
+        "populationType": "BodyOfWater",
+        "contaminatedThing": "GroundWater",
+        "contaminant": "inChIKey/HCHKCACWOHOZIP-UHFFFAOYSA-N",
+        "measuredProperty": "concentration"
+    },
+    "Cadmium": {
+        "populationType": "BodyOfWater",
+        "contaminatedThing": "GroundWater",
+        "contaminant": "Cadmium",
+        "measuredProperty": "concentration"
+    }
+}
+_INTERMEDIATE_CSV_NAME = "tar_creek_2020_corrected.csv"
+_CLEAN_CSV_FRAMES = []
```

Comment:
this isn't quite a constant, but is more of a global. it's not encouraged (harder to test, etc). can this be passed between the functions instead?

Reply:
Got it! I have changed the input flag to be the location of the file